### DISCUSSION TITLE
How to add our instance to https://joinmastodon.org/communities/general

### DISCUSSION THREAD

**Discussion by zaclys** [mastodon/mastodon#17578] [Q&A] [answered]
Hi dear Mastodon team,

sorry we're trying to ask here because our request doesn't seem to be received through the mastodon email.

The question is in the title but to give more details, here is the full request : 

Bonjour ! Hello,

As FOSS french provider, Zaclys freely provides Mastodon to its community : https://mastodon.zaclys.com/

We would like to know how to guide them to our instance through the official page : https://joinmastodon.org/communities ? Can you help us ?

It would be great to appear when filtered by french language.
And regarding categories, we don't focus on special topics as our instance is a general purpose social network alternative.
So we could appear everywhere when filtered by french language.

Thanks a lot for your help.

Best whishes,
The Zaclys team
https://www.zaclys.com/

**Comment by idanoo** [comment]
https://github.com/mastodon/joinmastodon/issues/93#issuecomment-383069745

see this answer

**Reply by Gargron** [reply]
That’s not accurate anymore (for many years). And zaclys had already been added.

**Reply by idanoo** [reply]
Ahh good to know - I was also looking and stumbled across it. Thanks 👍

**Comment by Gargron** [comment, marked as answer]
https://joinmastodon.org/covenant

**Reply by Lastorder-DC** [reply]
Looks like page is now removed(or moved). Any updates?
